src/block/horizontal-scroller/index.php

```diff
@@ -19,3 +19,38 @@ function stackable_load_horizontalscroller_frontend_script() {
 	}
 	add_action( 'stackable/horizontal-scroller/enqueue_scripts', 'stackable_load_horizontalscroller_frontend_script' );
 }
+
+if ( ! function_exists( 'stackable_horizontal_scroller_add_class_images' ) ) {
```

Comment:
```suggestion
// Lazy loaded images inside carousels make the carousel height buggy and show extra
// spaces because the height isn't available in lazy loaded images. Prevent images from
// hidden slides from lazy loading to prevent this.
if ( ! function_exists( 'stackable_horizontal_scroller_add_class_images' ) ) {
```